.github/workflows/ci.yml

```diff
@@ -166,6 +166,99 @@ jobs:
           path: |
             ~/.m2/repository/org/apache/ozone
           retention-days: 1
+  build-config-doc:
```

Comment:
We need conditionals so this only runs on pushes to the master branch in the main Ozone repo:
```
if: ${{  github.repository == 'apache/ozone' && github.event_name == 'push' && github.ref_name == 'master' }}
```